supporting_scripts/analysis-of-endpoint-connections/src/main/typeScript/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "commonjs",
+  "name": "analysis-of-endpoint-connections",
+  "version": "1.0.0",
+  "description": "Parses Typescript files and generates a report of all the endpoint connections in the file.",
+  "main": "AnalysisOfEndpointConnectionsClient.ts",
+  "license": "ISC",
+  "scripts": {
+    "start": "node -r esm AnalysisOfEndpointConnectionsClient.ts"
+  },
+  "dependencies": {
```

Comment:
The dependencies are defined separately for this subproject, even though they are defined in the parent project. This is because changes in the parent project's dependencies could else negatively affect the subproject.